docker/scripts/start-core.sh

```diff
@@ -0,0 +1,68 @@
+#script for initializing an alpine container with an injective release
+install_tools() {
+  apk update && apk upgrade
+  apk add --no-cache wget zip gcompat libgcc
+}
+
+get_latest_binary() {
+  wget $GIT_RELEASE_REPO/$GIT_TAG/$RELEASE_ARCH
+  unzip -o $RELEASE_ARCH
+  mv -f injectived peggo injective-exchange /usr/bin
+  mv -f libwasmvm.x86_64.so /usr/lib
+}
+
+injectived_set_working_dir() {
+  mkdir -p $HOME/.injectived-artifacts
+  cd $HOME/.injectived-artifacts
+}
+
+injectived_clean_working_dir() {
+  rm -rf $HOME/.injectived-artifacts
+  echo "### Provisioning Finished. Cleaning... ###"
+}
+
+injectived_start_testnet() {
+  export GIT_TAG=$GIT_TESTNET_TAG
+  export GIT_NETWORK_CONFIG=$TESTNET_CONFIG
+  export NETWORK_CONFIG_PATH=$TESTNET_CONFIG_LATEST
+  export GIT_RELEASE_REPO=$GIT_RELEASE_REPO_TESTNET
```

Comment:
`GIT_RELEASE_REPO_TESTNET` is missing in the provisioner container, [I see only mainnet](https://github.com/InjectiveLabs/injective-chain-releases/pull/40/files#diff-c661df96370174ec56fd0c88632fb41bef6f2af1f1946f118ea5c29469923ecbR14)

Reply:
It's on the [next line](https://github.com/InjectiveLabs/injective-chain-releases/blob/841326db6bd0d3d814a48f655691a7e41873a6d6/docker/addons/docker-compose.provisioner.yaml#L18). Click on the expand button :)